### ISSUE TITLE
train the caer dataset has a problem, cant run it successfully.

### ISSUE THREAD

**Issue by Issac304** [Sampson-Lee/DSCT#4]
caer has fail to train.
![image](https://github.com/user-attachments/assets/9115db7d-592c-464d-b0cf-6a5b0e44bf47)
and caer dataset no has the val annotation.
![image](https://github.com/user-attachments/assets/0f73badf-f9ed-46f4-a0f1-dd7cc20132eb)

**Comment by Issac304** [comment]
![image](https://github.com/user-attachments/assets/68a4f931-e98a-4086-972f-554de36f7192)

**Comment by Sampson-Lee** [comment]
At my first look, binary_flag should be modified as 0. We establish a multiple-class task in caer-s dataset. The output is an integer instead of a binary like emotic. I will confirm this by running the project.

**Comment by Issac304** [comment]
yes i try all propobility， doesnt work， “ I will confirm this by running the project.”think you！

**Comment by Sampson-Lee** [comment]
caer has fail to train: I have fixed this bug. Please try again.

caer dataset no has the val annotation: I disgard the validation set and report the best result on the test set (following the previous setting). If you want to run experiments on the validation set, you can generate one using the `./datasets/scripts/caer_to_coco.py`.

**Comment by Issac304** [comment]
1、in caer data set, the test.py isnt used?    2、the training of caer is from scatch or from the pth file of emotic?
